src/amo/utils/index.js

```diff
@@ -48,3 +50,16 @@ export const getCanonicalURL = ({
 |}): string => {
   return `${_config.get('baseURL')}${locationPathname}`;
 };
+
+export const isInternalURL = ({
+  _config = config,
+  urlString,
+}: {|
+  _config?: typeof config,
+  urlString: string,
+|}): boolean => {
+  const baseURL = _config.get('baseURL');
+  const urlParts = url.parse(urlString, true);
+
+  return !urlParts.protocol || baseURL.includes(urlParts.host);
```

Comment:
What if `urlString` is on a subdomain like `reviewers.addons.mozilla.org`? Will that be considered internal? It should not.

Reply:
Ah, good point, I guess I need to check if the host from `baseURL` is equal to the `host` from `urlParts`.